src/prodbg/main/src/windows.rs

```diff
@@ -700,6 +710,65 @@ impl Window {
             Self::remove_views(self, view_plugins, &views_to_delete);
             self.save_cur_workspace_state();
         }
+
+        self.render_view_rename_dialogue(view_plugins);
```

Comment:
I think dialogue here should be dialog. dialogue is more used in a conversation context while dialog is more common for a "computer dialog box"